src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,23 @@
 import React from 'react';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+
+type Props = {
+  movies: Movie[]
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(key => (
+      <MovieCard
```

Comment:
```suggestion
    {movies.map(movie => (
      <MovieCard
        key={movie.imdbId}
```
Не надо переименовывать сам ключ, тебе надо добавить отдельный атрибут key
https://reactjs.org/docs/lists-and-keys.html